### PR TITLE
Refactor db#DeleteClass into a handler function

### PR DESCRIPTION
**Pull request by VictorWen**
Added DeleteClass to the TLAB interface and implemented it in db,go and mock.go. Refactored db#DeleteClass from db/class.go into a handler function in handler/class.go. Added unit tests to class_test.go.

### CODE REVIEW

File: handler/class_test.go [hunk 1]
```diff
@@ -307,3 +307,92 @@ func TestGetClass(t *testing.T) {
...
+			TLADB:   d,
+		})) {
+			require.Equal(t, http.StatusOK, rec.Code)
+			_, err := d.LoadClass(context.Background(), "test")
+			require.Error(t, err)
+			_, err = d.LoadProgram(context.Background(), "test")
+			require.Error(t, err)
+		}
+	})
+}
```
Comment (lumisphere902) [inline handler/class_test.go:398]: nit: newline

File: db/class_test.go [hunk 2]
```diff
@@ -179,8 +179,8 @@ func TestCreateClass(t *testing.T) {
 	obj.D = ptr
 	require.NoError(t, err)
 
-	CreateTestUser(t, &obj, 0)
-	CreateTestClass(t, &obj, 0, 0)
+	//CreateTestUser(t, &obj, 0)
```
Comment (lumisphere902) [inline db/class_test.go:182]: Were these failing? Or is commenting out the delete calls enough

File: handler/class.go [hunk 3]
```diff
@@ -105,3 +107,46 @@ func GetClass(cc echo.Context) error {
...
+	if err := c.DeleteClass(c.Request().Context(), class.CID); err != nil {
+		if status.Code(err) == codes.NotFound {
+			return c.String(http.StatusNotFound, "could not find class")
+		}
+
+		return c.String(http.StatusInternalServerError, errors.Wrap(err, "failed to delete class").Error())
+	}
+
+	return c.String(http.StatusOK, "")
+}
```
Comment (lumisphere902) [inline handler/class.go:152]: nit: newline

**Review by lumisphere902 — CHANGES_REQUESTED** [review]
Mostly looks good; just some lint errors and a couple of small nits

File: db/class_test.go (hunk 2 above)
Comment (VictorWen) [reply db/class_test.go]: Yeah, no, they're all fine.

**Comment by VictorWen** [comment]
Added newlines at the end of handler/class.go and handler/class_test.go. Uncommented the create test cases in db/class_test.go

**Review by lumisphere902 — APPROVED** [review]
LGTM, and congrats on your first PR! One small note for the future: you can name your branches something like `issue66-{description}`, just so it's easier to see what the branch is for outside of the GitHub UI.

**Pull request merged**